db/migrate/20211105142202_queue_notification_update_mail.rb

```diff
@@ -1,5 +1,9 @@
 class QueueNotificationUpdateMail < ActiveRecord::Migration[6.1]
   def up
+    # On a newly created database, we don't want the update mail to be sent.
+    # Users are only created upon seeding.
+    return unless User.exists?
```

Comment:
```suggestion
    return unless User.not_builtin.exists?
```

do we have no previous migrations that some call on `User.system` or something? I'd probably add not_builtin just to be sure. LGTM otherwise 👍

Reply:
Currently we don't have any users in the system after all the migrations ran through. But admittedly, this might change so let's be careful.